frontend/src/index.css

```diff
@@ -115,3 +117,21 @@ input[type="number"] {
   background-color: hsl(76deg 55% 77%);
   color: hsl(84deg 55% 33%);
 }
+
+.save-button-container {
+  display: flex;
+  justify-content: end;
+}
+
+.accessibility-label {
+  display: none;
+}
+
+.plus-button {
+  flex: 0 0 auto;
```

Comment:
I learned just the other day that button elements can't be flex containers, so you can just remove this line. Here's a little bit of background: https://stackoverflow.com/questions/35464067/flex-grid-layouts-not-working-on-button-or-fieldset-elements

Reply:
sure thanks, a button should not be a flex container.